### ISSUE TITLE
converting obspy event catalog fails

### ISSUE THREAD

**Issue by trichter** [pyrocko/pyrocko#350]
Hi!

I am running recent versions of ObsPy and Pyrocko.

The following code fails.

```
In [15]: from obspy import read_events
In [16]: from pyrocko import obspy_compat
In [17]: obspy_compat.plant()
In [18]: c = read_events()
In [19]: c.to_pyrocko_events()
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-19-94924390163e> in <module>()
----> 1 c.to_pyrocko_events()

~/anaconda/envs/workhorse/lib/python3.6/site-packages/pyrocko/obspy_compat/base.py in to_pyrocko_events(catalog)
     99 
    100             events.append(model.Event(
--> 101                 name=obspy_event.resource_id + origin.resource_id,
    102                 time=origin.time.timestamp,
    103                 lat=origin.latitude,

TypeError: unsupported operand type(s) for +: 'ResourceIdentifier' and 'ResourceIdentifier'

In [20]: str(c[0].resource_id)
Out[20]: 'quakeml:eu.emsc/event/20120404_0000041'
```

I guess, converting the id attributes with str should be enough.

**Comment by HerrMuellerluedenscheid** [comment]
Hi @trichter,

Thanks for reporting this. It should be fixed now (see fc673e45be71c94d5c2e2249f451c571ae91e91b) . Yes, it was basically just converting to string.
Best regards
Marius

**Comment by trichter** [comment]
Thank you for fixing
